client/src/pages/stash/BusinessesFiltered copy.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
```

Comment:
Hi @cynthiaeddy! Is this duplicate file needed?

Reply:
nope, deleted, will commit then merge